### DISCUSSION TITLE
.NET 7 support

### DISCUSSION THREAD

**Discussion by dsolteszopyn** [fluentassertions/fluentassertions#2041] [General]
When can we expected .net 7 support?

**Comment by eNeRGy164** [comment]
Maybe you can explain what you are looking for? Is there something not working for you?

**Reply by dsolteszopyn** [reply]
Nothing is needed to use this package in .net 7?  No dependencies etc?  .net 6 package will work fine in .net 7 project?

**Reply by jnyrup** [reply]
yes, yes and yes.

I verified months ago that Fluent Assertions works with .NET 7.
